### ISSUE TITLE
Donkey Kong Country Tropical Freeze

### ISSUE THREAD

**Issue by darkuser89** [SamoZ256/cemu-metal-game-compatibility#4]
## Description
Crash
Emulator crashes when collecting an object
[log.txt](https://github.com/user-attachments/files/17732043/log.txt)
[log Kopie.txt](https://github.com/user-attachments/files/17732047/log.Kopie.txt)

Describe the issues with the game (`Perfect` if none).

## Screenshots
<img width="1392" alt="Bildschirmfoto 2024-11-13 um 13 00 04" src="https://github.com/user-attachments/assets/4cd2a937-8858-4f3d-8207-4fdf14b27f3d">

Please include at least one screenshot of the game (if possible).

## Cemu Metal version
testing1.0.2

Put the version of Cemu you used for testing (e.g. `testing1.0.0`).

## Similar reports

- [X] I have checked that the game hasn’t been reported on the `Issues` page yet (please tick the checkbox by writing `X` inside `[ ]`)

**Comment by SamoZ256** [comment]
Oh, ok. I see that it failed to compile some shaders, but that shouldn't crash.

**Comment by darkuser89** [comment]
test on 1.0.9 = Same Crash

[log.txt](https://github.com/user-attachments/files/18117261/log.txt)

**Comment by SamoZ256** [comment]
@darkuser89 could you please test [this build](https://drive.google.com/file/d/1Ci8p8lVKTHdmo_Ydp7yBzs_uHycjWf51/view?usp=sharing)? I hope it fixes the crash and there won't be any visual issues when collecting an object. Also, did you have to somehow configure this game for it to boot? Cause it crashes for me after the splash screen

**Comment by darkuser89** [comment]
no, I have everything on standard. the game starts with the test build and the objects can be collected

https://github.com/user-attachments/assets/db886a01-11a1-4489-aafb-70f280b4216c

testing2.3.1 and 2.3.0 = crash on boot 

EDIT:

on my M3 Pro / 15.2 and M1 Max / 15.2 The test build starts without any problems

**Comment by SamoZ256** [comment]
Great! Have you noticed any other issues?

**Comment by darkuser89** [comment]
<img width="1341" alt="Image" src="https://github.com/user-attachments/assets/1311d84e-aeb3-4dd8-a0c8-811236835545" />
<img width="1341" alt="Image" src="https://github.com/user-attachments/assets/82f0a22f-4773-4610-8d31-967503fd8260" />

Unfortunately not in level 1-4 there is shader explosion

**Comment by SamoZ256** [comment]
@darkuser89 got the game to work (well actually, it worked right away, I just haven't tried it in a long time). I have encountered such vertex explosions on the title screen, but on second startup everything was fine, indicating that it was probably an issue with async shader compilation. Could you please try once more to see if that's the case?

**Comment by darkuser89** [comment]
yes after restarting the game the level looks normal again
